homeassistant/components/route53/__init__.py

```diff
@@ -36,6 +38,7 @@
                 vol.Required(CONF_SECRET_ACCESS_KEY): cv.string,
                 vol.Required(CONF_ZONE): cv.string,
                 vol.Optional(CONF_TTL, default=DEFAULT_TTL): cv.positive_int,
+                vol.Optional(CONF_URL, default=DEFAULT_URL): cv.string,
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configuration or changes.

This integration needs to be refactored first to use a configuration flow and config entries first.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

As these changes often involve a bit of work and some significant shift in the current code, we will close this PR for now.

We (and our community!) would really appreciate it if you could start on the refactoring of this integration in a new PR.

Thanks already! :+1:

Reply:
@frenck, if I removed the second commit (the commit with the behavioural change), would you be willing to accept just the test suite I added? The tests may be helpful for future refactoring.

Reply:
You are absolutely welcome to open up a new PR to extend the test suite, sure 👍